erpnext/www/lms/topic.py

```diff
@@ -61,4 +61,22 @@ def calculate_contents_progress(contents):
 		if content.get("completed"):
 			completed_content = completed_content + 1
 	total_progress = int((completed_content * 100) / total_content)
-	return total_progress
\ No newline at end of file
+	return total_progress
+
+def get_ongoing_topic(contents):
+	ongoing_content = ""
+	ongoing_topics = []
+	ongoing_topics_length = 0
+	total_content = len(contents)
+	for content in contents:
+		if not content.get("completed"):
+			ongoing_topics.append(content)
+
+	ongoing_topics_length = len(ongoing_topics)
+	if total_content != ongoing_topics_length:
+		for ongoing_topic in ongoing_topics:
+			ongoing_content = ongoing_topic.get('content')
+			break
```

Comment:
You can directly return ongoing_topic[0].get('content') since ongoing_topic is a list of incomplete topics